doc/parameter.rst

```diff
@@ -121,10 +121,12 @@ Parameters for Tree Booster
 * ``lambda`` [default=1, alias: ``reg_lambda``]
 
   - L2 regularization term on weights. Increasing this value will make model more conservative.
+  - range: [0,∞]
```

Comment:
Thank you for the improvement, could you please consider using a LaTeX symbol with `:math:`?

Reply:
it's `:math:` ,not `math:`.

Reply:
Thank you for your patience 😅